### PR TITLE
[DOC] Consolidate BERT fine-tuning results in a table

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
As title. Preview: https://github.com/eric-haibin-lin/gluon-nlp/blob/table/scripts/bert/index.rst 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-613/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-613/4/index.html

**Comment by mli** [comment]
Job PR-613/6 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-613/6/index.html

**Comment by mli** [comment]
Job PR-613/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-613/7/index.html

**Comment by StrayBird-ATSH** [comment]
Currently I am writing a page of all the models available with their model sources. But perhaps this part has involved a lot of articles and for me it is a little difficult to clearly distinguish which article is linked to which model so that I couldn't use footnotes to write in the new web page.
Is it possible that you could help give some more intuitive words in the webpage about the article source?
#1184 
Thanks!

**Comment by eric-haibin-lin** [comment]
@StrayBird-ATSH can you leave comments in the PR for the ones/references you are not sure?

File: scripts/bert/index.rst [hunk 1]
```diff
@@ -23,43 +29,144 @@ BERT for Sentence Classification on GLUE tasks
...
+|                     |                                                  MRPC                                                  |                                                  RTE                                                  |                                                 SST-2                                                 |                                                MNLI-m/mm                                               |
++---------------------+--------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------+
```
Comment (StrayBird-ATSH) [inline scripts/bert/index.rst:34]: Could you please help provide references for the models in this table? Thanks 
#1184

File: scripts/bert/index.rst [hunk 2]
```diff
@@ -23,43 +29,144 @@ BERT for Sentence Classification on GLUE tasks
...
+|                       |                                                            SQuAD 1.1                                                            |                                                             SQuAD 1.1                                                            |                                                             SQuAD 2.0                                                            |
++-----------------------+---------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+
```
Comment (StrayBird-ATSH) [inline scripts/bert/index.rst:92]: Could you please help provide references for the models in this table? Thanks
#1184

File: scripts/bert/index.rst (hunk 1 above)
Comment (eric-haibin-lin) [reply scripts/bert/index.rst:34]: https://arxiv.org/abs/1810.04805

File: scripts/bert/index.rst (hunk 2 above)
Comment (eric-haibin-lin) [reply scripts/bert/index.rst:92]: https://arxiv.org/abs/1810.04805

**Pull request merged**